test/gke/select-cluster.sh

```diff
@@ -36,7 +36,7 @@ while [ $locked -ne 0 ]; do
 
 	cluster_name=${cluster_uri##*/}
 	echo "checking whether cluster ${cluster_uri} has a running operation"
-	if gcloud container operations list | grep RUNNING | grep "${cluster_name}" ; then
+	if $(gcloud container operations list --project "${project}" --filter="status=RUNNING AND targetLink=${cluster_uri}") ; then
 		echo "cluster has an ongoing operation, trying out another cluster"
```

Comment:
This line will always run, `gcloud` doesn't return error on empty list:

```
$ if $(gcloud container operations list --project "cilium-ci" --filter="status=RUNNING AND targetLink=trolo"); then echo "true"; fi
Listed 0 items.
true

```

Reply:
Yea I did see that xD Fixed in latest commit.

Reply:
I don't think that's fixed, I tested this locally:

```
$ if $(gcloud container operations list --project "cilium-ci" --filter="status=RUNNING AND targetLink=trolo" --format="value(name)"); then echo "true"; fi
true
```

Reply:
Right, cause I can't Bash.